qmcpy/algorithms/function/asian_call_integrand.py

```diff
@@ -0,0 +1,50 @@
+''' Originally developed in MATLAB by Fred Hickernell. Translated to python by Sou-Cheng T. Choi and Aleksei Sorokin '''
```

Comment:
Putting historical comments in the code is odd -- generally the code is just considered alive and the GitHub history records how it got to where it is.  If you want to make a permanent historical statement, putting it in a separate text file elsewhere would be preferable (as this file could be replaced by something else tomorrow).

Reply:
Also, docstrings should be `"""` not `'''`

Reply:
The line ''' Originally developed in MATLAB ...''' has been removed from all files in the most recent commit. 
As we go through and make files PEP8 compatible we'll change ''' to """
Files in algorithms/accum_data and algorithms/stop have been updated to have better docstrings, variable names (lower case), and google styling. 
Throughout the week we'll get the rest of the files looking nice